client/src/components/Register.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom';
+import {connect} from 'react-redux';
+import {register} from '../actions/index';
+
+class Register extends React.Component{
+    
+    constructor(props){
+        super(props);
+        this.state = {
+            username: '',
+            password1: '',
+            password2: '',
+        }
+    }
+
+    handleInput = event => {
+        event.preventDefault();
+        this.setState({
+            [event.target.name]: event.target.value
+        })
+    }
+
+    handleRegister = event => {
+        event.preventDefault();
+        const newUser = {
+            username: this.state.username,
+            password1: this.state.password1,
+            password2: this.state.password2
+        }
+        this.props.register(newUser);
+    }
```

Comment:
Why doesn't this load the game screen after registration? Seems like that might be desirable beahviour, like you have for the login.

Reply:
I had cut the registration redirect while attempting to implement the redirect in another place (actions/reducers), so that the pusher registration could propagate properly on the redirect. I was unable to find a solution in time for the demos, so it just never got put back in. It's back for now, though there's a package (react-router-redux) that may offer some insight as to how to implement redux redirects that I'm looking into.